### DISCUSSION TITLE
how to generate a random payload in http responses using Trex Cisco Traffic generator

### DISCUSSION THREAD

**Discussion by Petrov-Daniil** [cisco-system-traffic-generator/trex-core#1084] [Q&A]
I found this exmaple https://trex-tgn.cisco.com/trex/doc/trex_astf.html#_tutorial_wrapping_it_up_example in the official documentation, but I don’t understand how it works and whether it can somehow be in astf mode adapted via HTTP traffic so that the server sends a request and receives a different response each time.

I tried to run the example from the documentation, checked the API doc but did not find the methods and classes I needed

**Comment by hhaim** [comment]
@Petrov-Daniil just answer this in the gmail
